### PR TITLE
Changes to accept null gender race and ethnicity

### PR DESCRIPTION
**Pull request by luis1van**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by dczulada** [comment]
Looks like we need to do some work in DemographicsRandomizer.

If we use Vendor Patients (in a CVU+ test) that don't have a gender, it will break.

File: lib/cypress/demographics_randomizer.rb [hunk 1]
```diff
@@ -142,5 +142,18 @@ def self.sample_payer(patient, prng)
       end
       payer_hash
     end
+
+    # work around for null gender | race | ethnicity
+    def self.assign_default_demographics(patient)
+      elements = []
+      elements << QDM::PatientCharacteristicSex.new(dataElementCodes: [{ 'code' => 'M', 'codeSystem' => 'NA' }]) unless patient&.gender
```
Comment (dczulada) [inline lib/cypress/demographics_randomizer.rb]: Lets use real codes
Gender: code: 'M' codeSystem: '2.16.840.1.113883.5.1'
race: code: '2028-9 codeSystem: '2.16.840.1.113883.6.238'
ethnicity: code: '2186-5' codeSystem: '2.16.840.1.113883.6.238'
payer: code: '1' codeSystem: '2.16.840.1.113883.3.221.5'

File: lib/ext/patient.rb [hunk 2]
```diff
@@ -96,11 +96,8 @@ def payer
 
     def gender
       gender_chars = qdmPatient.get_data_elements('patient_characteristic', 'gender')
-      if gender_chars&.any? && gender_chars.first.dataElementCodes &&
-         gender_chars.first.dataElementCodes.any?
+      if gender_chars&.any? && gender_chars.first.dataElementCodes && gender_chars.first.dataElementCodes.any?
         gender_chars.first.dataElementCodes.first['code']
-      else
-        raise 'Cannot find gender element'
```
Comment (dczulada) [inline lib/ext/patient.rb]: Looks like we also need to remove this for payer

File: lib/cypress/demographics_randomizer.rb [hunk 3]
```diff
@@ -142,5 +142,18 @@ def self.sample_payer(patient, prng)
...
+      elements << QDM::PatientCharacteristicRace.new(dataElementCodes: [{ 'code' => 'NA', 'codeSystem' => 'NA' }]) unless patient&.race
+      elements << QDM::PatientCharacteristicEthnicity.new(dataElementCodes: [{ 'code' => 'NA', 'codeSystem' => 'NA' }]) unless patient&.ethnicity
+      unless patient&.payer
+        elements << QDM::PatientCharacteristicPayer.new(dataElementCodes: [{ 'code' => '1', 'codeSystem' => 'NA' }],
+                                                        relevantPeriod: QDM::Interval.new(patient.birthDatetime, nil))
```
Comment (dczulada) [inline lib/cypress/demographics_randomizer.rb]: this should be patient.qdmPatient.birthDatetime
